.project

```diff
@@ -14,4 +14,15 @@
 	<natures>
 		<nature>org.eclipse.jdt.core.javanature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
Changes in <filteredResources> are probably due to change in IDE, the code corresponds to Eclipse but I have it opened in VSCode which is the cause.